lib/pbench/server/indexing_tarballs.py

```diff
@@ -320,7 +340,11 @@ def sigquit_handler(*args):
                                 failures,
                                 retries,
                             )
-                            tb_res = 1 if failures > 0 else 0
+                            tb_res = (
+                                error_code["OP_ERROR"]
+                                if failures > 0
+                                else error_code["OK"]
+                            )
```

Comment:
For your consideration...since it's only the error which is the conditional,
```
                            tb_res = error_code["OP_ERROR" if failures > 0 else "OK"]
```

Reply:
done